service/lib/dinstaller/config.rb

```diff
@@ -28,41 +30,78 @@ module DInstaller
   # This also means that config needs to be re-evaluated if conditions
   # data change, like if user pick different distro to install.
```

Comment:
Dummy question: still that documentation true?

Reply:
Well, the conditions are still something pending to implement and at least the new product selection config should be merged in current one... So, did not considered to reword it by now ;)